Content.IntegrationTests/Tests/GameRules/NukeOpsTest.cs

```diff
@@ -50,6 +50,10 @@ public async Task TryStopNukeOpsFromConstantlyFailing()
         var invSys = server.System<InventorySystem>();
         var factionSys = server.System<NpcFactionSystem>();
 
+        // test urist is a noob let him be nukie
+        Assert.That(server.CfgMan.GetCVar(CCVars.GameRoleTimers), Is.True);
```

Comment:
Ah, apparently its already disabled in tests by default

Reply:
bruh????